Source/Scene/Cesium3DTileFeatureTableResources.js

```diff
@@ -0,0 +1,80 @@
+/*global define*/
+define([
+        '../Core/ComponentDataType',
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/DeveloperError'
+    ], function(
+        ComponentDataType,
+        defaultValue,
+        defined,
+        DeveloperError) {
+    'use strict';
+
+    /**
+     * @private
+     */
+    function Cesium3DTileFeatureTableResources(featureTableJSON, featureTableBinary) {
+        this.json = featureTableJSON;
+        this.buffer = featureTableBinary;
+        this._cachedArrayBufferViews = {};
+        this.featuresLength = 0;
+    }
+
+    Cesium3DTileFeatureTableResources.prototype.getTypedArrayForSemantic = function(semantic, byteOffset, componentType, count) {
+        //>>includeStart('debug', pragmas.debug);
+        if (!defined(byteOffset)) {
+            throw new DeveloperError('byteOffset must be defined to read from binary data for semantic: ' + semantic);
+        }
+        if (!defined(componentType)) {
+            throw new DeveloperError('componentType must be defined to read from binary data for semantic: ' + semantic);
+        }
+        if (!defined(count)) {
+            throw new DeveloperError('count must be defined to read from binary data for semantic: ' + semantic);
+        }
+        //>>includeEnd('debug');
+        var cachedArrayBufferViews = this._cachedArrayBufferViews;
+        var arrayBuffer = cachedArrayBufferViews[semantic];
+        if (!defined(arrayBuffer)) {
+            arrayBuffer = ComponentDataType.createArrayBufferView(componentType, this.buffer, byteOffset, count);
+            cachedArrayBufferViews[semantic] = arrayBuffer;
+        }
+        return arrayBuffer;
+    };
+
+    Cesium3DTileFeatureTableResources.prototype.getGlobalProperty = function(semantic, componentType, count) {
```

Comment:
I don't think we should assume it will always be JSON.  A global property could be a big array, even if it isn't per-feature data, so we shouldn't make the spec this limited unless there is a significant implementation reason, which I do not see here since we already need to handle binary for the other properties.